theme_zap/views/snippets/s_striped_top.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_striped_top" inherit_id="website.s_striped_top">
```

Comment:
It would be better a different combination of colors to increase readability 
![image](https://github.com/user-attachments/assets/381e892a-344c-41a2-993a-cb55201eba06)